internal/controller/alert_rule.go

```diff
@@ -119,20 +122,28 @@ func ParseAlertRule(tier, service string, in monitoringv1.Rule) ([]monitoringv1.
 		return nil, nil
 	}
 
+	// Default labels
+	lab := map[string]string{
+		"severity": "info",
+		"playbook": "https://git.io/absent-metrics-operator-playbook",
```

Comment:
Do we really need a short link here? https://github.com/sapcc/absent-metrics-operator/blob/master/doc/playbook.md is not that much longer and more understandable.

Reply:
Changed.